src/figuresClasses.ts

```diff
@@ -1,19 +1,110 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
 export class Triangle {
+  shape: string;
+
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  color: Figure;
+
+  constructor(
+    color: Figure,
+    a: number,
+    b: number,
+    c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('your error message');
+    }
 
+    if (a >= b + c || b >= a + c || c >= a + b) {
+      throw new Error('your error message');
```

Comment:
Provide a specific error message like 'Longest side of a triangle should be less than the sum of the other two sides'